labman/db/tests/test_process.py

```diff
@@ -1573,6 +1575,13 @@ def test_generate_prep_information(self):
         exp = {Study(1): SHOTGUN_EXAMPLE}
         self.assertEqual(obs[Study(1)], exp[Study(1)])
 
+    # Ok, the Knight lab doesn't test private methods.  But I do :) AEB
```

Comment:
Should this block be uncommented, as per the note? 😄

Reply:
Urgh, I suppose I can't just summarily change the testing procedure, so I'll remove the private test.